FBC/ProductsAndServices/ClientsAndAccounts.rdf

```diff
@@ -164,11 +192,17 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>account identifier</rdfs:label>
-		<skos:definition>an identifier that identifies an account</skos:definition>
+		<skos:definition>identifier that denotes an account</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom>ISO 13616-1:2007 Financial services - International bank account number (IBAN)</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:synonym>account number</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fbc-pas-caa;AccountOpenDate">
```

Comment:
As for AccountCloseDate, I don't see why AccountOpenDate needs a specific class.

Reply:
See above - most banks will need the reified class for managing account information and mapping to their various source systems, but I've removed these and am using ExplicitDate until we have a use case / request.